include/config/overworld.h

```diff
@@ -44,8 +44,10 @@
                                                   // 16x32, 32x32, 64x64 etc are fine
 // Follower Pokémon
 #define OW_FOLLOWERS_ENABLED           FALSE      // Enables follower Pokémon, HGSS style. Requires OW_POKEMON_OBJECT_EVENTS. Note that additional scripting may be required for them to be fully supported!
-#define OW_FOLLOWERS_BOBBING           TRUE       // If true, follower pokemon will bob up and down during their idle & walking animations
-#define OW_FOLLOWERS_POKEBALLS         TRUE       // Followers will emerge from the pokeball they are stored in, instead of a normal pokeball
+#define OW_FOLLOWERS_BOBBING           TRUE       // If TRUE, follower Pokémon will bob up and down during their idle & walking animations
+#define OW_FOLLOWERS_POKEBALLS         TRUE       // If TRUE, follower Pokémon will emerge from the Poké Ball they are stored in, instead of a normal Poké Ball
+#define OW_FOLLOWERS_WEATHER_FORMS     FALSE      // If TRUE, follower Pokémon that transform in battle via weather will also transform in the overworld.
```

Comment:
Forms still change in the party, even with this config disabled
![image](https://github.com/user-attachments/assets/9774a902-5b86-4e28-afe5-bab576c2384c)

Reply:
I think we may need a config to differentiate between them only changing appearance in the overworld (which is purely cosmetic, like `OW_FOLLOWERS_COPY_WILD_PKMN`) and one that actually changes the form (since this causes non-vanilla interactions when it enters a battle with starting entry hazards)

Reply:
Doesn't even have to be starting hazards. Actually the first mon won't be affected by it but the transformed mons in the bag will which would be solved by:

> This should only affect the Pokémon in the first slot of the party (the one currently following, rather). It's the fact they're out of the ball that makes them change forms. As a result, I think all other species should probably revert back to "normal" conditions if they're not currently outside of the ball.